src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1442,6 +1442,7 @@ inlines1 = mconcat <$> many1 inline
 
 inline :: PandocMonad m => MarkdownParser m (F Inlines)
 inline = choice [ whitespace
+                , githubWikiLink
```

Comment:
Is there a reason this is here and not further down in the list?
Probably best to put it right before the regular link and image parsers; that should improve performance a bit.